user/views/profile.py

```
@@ -185,17 +186,53 @@ def post(self, request, *args, **kwargs):
         "page_title": "Beta profile",
-        "content": "Profile Beta content"
+        "content": "Profile Beta content",
+        "profile": profile,
+        "Header": "Profile",
+        # "experiences": experiences,
+        "letter": letter.content,
+        "contact_profiles": contact_profiles.filter(privacy=0),
+        "qr_code_base64": qr_code_base64,
+        "public_profile_url": public_profile_url,
+        "privacy_choices": privacy_choices,
+        "contact_profiles": contact_profiles,
```

Ruff (0.11.9)

Dictionary key literal `"contact_profiles"` repeated

(F601)